src/common/base_client.ts

```diff
@@ -260,6 +265,19 @@ export abstract class BaseClient {
         }
     }
 
+    protected async handleNodeNotifyRequest(request: NodeNotifyRequest): Promise<void> {
+        const nodeNotifyRequest: any = request;
+
+        try {
+            const response = new DefaultKolibriResponse(request.id, 0);
+            await this.connection.sendResponse(response);
+            this.subscription.onNodeNotify?.(nodeNotifyRequest.params);
```

Comment:
Can nodeNotifyRequest be assumed to have params, when its an any type?

Reply:
NodeNotifyRequest object has a params object

Reply:
with any type it can be assumed to have any sort of objects in it. Changed it to NodeNotifyRequest which has a params object